mne/time_frequency/tests/test_multitaper.py

```diff
@@ -3,7 +3,7 @@
 from numpy.testing import assert_array_almost_equal
 from distutils.version import LooseVersion
 
-from mne.time_frequency import dpss_windows, multitaper_psd
+from mne.time_frequency.multitaper import dpss_windows, _psd_multitaper
```

Comment:
ok, I'll swap it for psd_multitaper and will just create a RawArray object.

As an aside - I've got some helper functions for myself that just create random data and turn it into either a RawArray or EpochsArray object. I find it useful for quickly debugging etc. Happy to add it as a private function or something if it's useful at all.